src/JackMD/ScoreHud/data/DataManager.php

```diff
@@ -119,6 +119,22 @@ public function getPlayerFaction(Player $player): string{
 		}
 		return "Plugin not found";
 	}
+	/**
+	 * @param Player $player
+	 * @return string
+	 */
+	public function getFactionPower(Player $player){
+		/** @var FactionMain $factionsPro */
+		$factionsPro = $this->plugin->getServer()->getPluginManager()->getPlugin("FactionsPro");
+		if($factionsPro instanceof FactionMain){
+			$factionName = $factionsPro->getPlayerFaction($player->getName());
+			if($factionName == null){
```

Comment:
Use `===` instead of `==`

Reply:
Wait I am closing this one, I need to make this better give me 5 minutes lol